database/migrations/2024_09_19_125543_add_default_tab_to_users.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table): void {
+            $table->string('default_tab')->default('feed')->after('mail_preference_time');
```

Comment:
Can you keep the default as `following`? based on this (https://github.com/pinkary-project/pinkary.com/issues/455)

Reply:
Why do you want to do it later? It's easy to do it here no?

Reply:
I have to make changes in the view layer for this implementation causing the PR becoming big in size. If we merge this one fist, then the view changes can be updated. Ao thats why seperate PR.